contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6191,9 +6223,13 @@ transformSelectIntoStmt(CreateTableAsStmt *stmt)
 	}
 
 	result = lappend(result, stmt);
+	
 	if (altstmt)
 		result = lappend(result, altstmt);
 
+	if (babel_4926_original_name_list != NIL)
```

Comment:
I think this additional variable is not needed, we can directly append into `result` if needed at line number `6195`.
[Bonus] In fact we can take this opportunity to even remove the redundant `altstmt` variable as well and directly append into `result` whenever needed.

Reply:
It won't work as we are first appending the SelectInto statement commands and after that appending the altstmt or newstmt. if we directly append it to result then the newstmt will be in queue before selectInto commands and it will throw an error while executing(it won't find the new table that we have created using the select into statement)